### DISCUSSION TITLE
Configurare dispositivo all'accensione con HASSIO/Deconz

### DISCUSSION THREAD

**Discussion by MarkIV4** [indomus/forum#967] [HUB: Altro (Homey, iHost, SmartThings, Fibaro ecc.)]
Ciao, qualcuno è riuscito a configurare su Deconz  lo stato che il dispositivo deve assumere all'accensione? 
Es. Va via la corrente, quando ritorna, il dispositivo/lampadina deve rimanere spento.
Sul sito di deconz non trovo niente riguardo a questo e alle configurazioni di un entità light.
Grazie

**Comment by indomus** [comment]
Devi farlo a livello di device, oppure con un'automazione di Home Assistant.

**Reply by MarkIV4** [reply]
Si ma sul sito di deconz non trovo quale sia la variabile da usare per farlo a livello di device. Ho letto la guida dove c'è l'esempio della durata del sensore di movimento ma riguardo la mia domanda non ho trovato niente
